packages/block-library/src/quote/edit.js

```diff
@@ -10,28 +10,43 @@ import { __ } from '@wordpress/i18n';
 import {
 	AlignmentToolbar,
 	BlockControls,
+	InnerBlocks,
 	RichText,
 	useBlockProps,
 } from '@wordpress/block-editor';
 import { BlockQuotation } from '@wordpress/components';
 import { createBlock } from '@wordpress/blocks';
+import { useSelect } from '@wordpress/data';
 
 export default function QuoteEdit( {
 	attributes,
 	setAttributes,
 	isSelected,
-	mergeBlocks,
-	onReplace,
 	className,
 	insertBlocksAfter,
+	clientId,
 } ) {
-	const { align, value, citation } = attributes;
+	const { align, citation } = attributes;
 	const blockProps = useBlockProps( {
 		className: classnames( className, {
 			[ `has-text-align-${ align }` ]: align,
 		} ),
 	} );
 
+	const { isAncestorOfSelectedBlock } = useSelect(
```

Comment:
We don't need to destructure here if we just return the result of `hasSelectedInnerBlock` directly, instead of wrapped in an object.